beacon-chain/p2p/BUILD.bazel

```diff
@@ -9,6 +9,7 @@ go_library(
         "dial_relay_node.go",
         "discovery.go",
         "doc.go",
+        "fork.go",
```

Comment:
Can you add ssz marshaler functions?

```starlark
ssz_gen_marshal(
    name = "ssz_generated_files",
    srcs = ["fork.go"],
    objs = [
       "EnrForkID",
    ],
)
```

Then add `:ssz_generated_files` as a source to this go library. 

This will make ssz.Marshal faster.

Reply:
+1